test/tests/concrete_moisture_heat_transfer/maqbeth_1d.i

```diff
@@ -40,10 +40,6 @@
     order = CONSTANT
     family = Monomial
   []
-  [moisture_capacity]
```

Comment:
The problematic grizzly testing only runs the problems in the assesment directory, so you don't need any of these changes. Please remove the changes for anything that touches the 'test' directory and resubmit.